app/lib/main.dart

```diff
@@ -1,6 +1,7 @@
 // ignore_for_file: prefer_const_constructors
 
 import 'dart:typed_data';
+//import 'package:device_preview/device_preview.dart';
```

Comment:
what is all this outcommented stuff? I suppose it shouldn't be in this PR...

Reply:
I added the device preview package when error fixing. it was left as a comment for future use. ok, I will remove it.